content/3_Silver/Two_Pointers.mdx

```diff
@@ -280,3 +280,72 @@ public class Books {
 ## Problems
 
 <Problems problems="ad" />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    What's the two pointers algorithm?
+    <Quiz.Answer correct>
+      Keeping track of two pointers to find a pair of indices which satisfy some condition.
+      <Quiz.Explanation>
+        Correct.
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      A way to answer range sum queries in $\mathcal{O}(1)$ time.
+      <Quiz.Explanation>
+        Incorrect. You might be thinking of [Prefix Sums](https://usaco.guide/silver/prefix-sums), another silver topic.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      An algorithm which splits an array into two halves to efficiently find an element satisfying a certain condition.
+      <Quiz.Explanation>
+        Incorrect.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    When would a solution not be possible?
+    <Quiz.Answer>
+      If the array isn't sorted.
+      <Quiz.Explanation>
+        Incorrect.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      If the pointers overlap.
+      <Quiz.Explanation>
+        Correct. When the pointers overlap, they're looking for another solution to test when it doesn't exist.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      If the pointers go out of the bounds of the array.
+      <Quiz.Explanation>
+        Incorrect.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following isn't true?
+    <Quiz.Answer>
+      The time complexity of a standard two pointers algorithm is $\mathcal{O}(N)$
+      <Quiz.Explanation>
+        Incorrect. Assuming that both pointers move one step at a time and the array isn't sorted, both pointers would take a maximum of $N$ steps.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      The time complexity of a standard two pointers algorithm is $\mathcal{O}(1)$
```

Comment:
```suggestion
      The space complexity of a standard two pointers algorithm is $\mathcal{O}(1)$
```

From your explaination, I would guess this should say "space"?

Reply:
Also, this isn't really true. You should clarify in the question statement wether you are including the array / sorting as part of the memory / time complexity